src/com/xilinx/rapidwright/design/DesignTools.java

```diff
@@ -2820,6 +2820,18 @@ public static void createMissingStaticSitePins(BELPin belPin, SiteInst si, Cell
                     net = si.getDesign().getGndNet();
                 }
             }
+            String belName = belPin.getBELName();
+            if (belName.endsWith("LUT")) {
+            	if (belName.charAt(1) == '6') {
+            		// Anything placed on a LUT6 does not require A6 to be tied to VCC
+            		return;
+            	}
+            	String lut6Name = belName.replace('5', '6');
+            	// Thus it can only be a LUT5
+            	if (belName.charAt(1) != '5') {
+            		throw new RuntimeException("Expected " + lut6Name + " instead of " + belName);
```

Comment:
```suggestion
            	// Thus it can only be a LUT5
            	if (belName.charAt(1) != '5') {
            		String lut6Name = belName.replace('5', '6');
            		throw new RuntimeException("Expected " + lut6Name + " instead of " + belName);
```